static/config/status-chain-genesis.json

```diff
@@ -0,0 +1,31 @@
+{
+  "alloc": {
+     "AdAf150b905Cf5E6A778E553E15A139B6618BbB7": {
+      "balance": "0x10F0CF064DD59200000"
+    },
+    "65C01586aa0Ce152835c788aCe665e91Ab3527b8": {
+      "balance": "0x1B1AE4D6E2EF500000"
+    }
+  },
+  "coinbase": "0x0000000000000000000000000000000000000000",
+  "config": {
+    "byzantiumBlock": 1700000,
+    "chainId": 777,
+    "daoForkSupport": true,
+    "eip150Block": 0,
+    "eip150Hash": "0x41941023680923e0fe4d74a34bdac8141f2540e3ae90623718e47d66d1ca4a2d",
+    "eip155Block": 10,
+    "eip158Block": 10,
+    "ethash": {},
+    "homesteadBlock": 0
+  },
+  "difficulty": "0x100000",
+  "extraData": "0x537461747573436861696e20507269766174652054657374696e67204e6574776f726b",
```

Comment:
What is this?

Reply:
extraData contains a hex-string: "StatusChain Private Testing Network" :)

Reply:
Ok, sounds good :)